pkg/features/features.go

```diff
@@ -51,8 +51,8 @@ var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
 	// Example: {Default: true, PreRelease: featuregate.Beta},
 }
 
-func SetFeatureGateDuringTest(tb testing.TB, f featuregate.Feature, value bool) func() {
-	return featuregatetesting.SetFeatureGateDuringTest(tb, utilfeature.DefaultFeatureGate, f, value)
+func SetFeatureGateDuringTest(tb testing.TB, f featuregate.Feature, value bool) {
+	featuregatetesting.SetFeatureGateDuringTest(tb, utilfeature.DefaultFeatureGate, f, value)
```

Comment:
I also can't find where this function is being used. Maybe we can just remove it.

Reply:
It was added so that when we use features we can use this.

Reply:
That being said, it’s not hard to add back. @ahg-g ?

Reply:
It was added in this PR https://github.com/kubernetes-sigs/jobset/pull/557/files. At that time, the feature gate feature was supported, but this feature has not been used in the current project. If it needs to be deleted, I think it is reasonable.

Reply:
it is fine, we can keep it.